libs/domain/site/src/services/price-mode/state/events.ts

```diff
@@ -0,0 +1,5 @@
+import { QueryEvent } from '@spryker-oryx/core';
+
+export const PriceModeChanged = 'oryx.PriceModeChanged';
+
+export type PriceModeChanged = QueryEvent<string>;
```

Comment:
Do we use it?

Reply:
Yes, is the event watching when user change's price mode